ui/pages/Project/components/PhenotypePrioritizedGenes.jsx

```diff
@@ -9,6 +9,7 @@ import { BaseVariantGene } from 'shared/components/panel/variants/VariantGene'
 import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
 import { loadPhenotypeGeneScores } from '../reducers'
 import { getPhenotypeDataLoading, getIndividualPhenotypeGeneScores } from '../selectors'
+import {PHENOTYPE_PRIORITIZATION_FIELDS} from "../constants";
```

Comment:
this should be failing linting - make sure to check that there are no linter errors before putting up the final PR?